src/js/components/action.jsx

```diff
@@ -62,6 +62,11 @@ export class ActionComponent extends Component {
     render() {
         const publicKeys = store.getState().app.publicKeys;
 
+        let style = {};
+        if (this.props.accentColor) {
+            style.backgroundColor = style.borderColor = `#${this.props.accentColor}`;
```

Comment:
It's the app maker's responsibility to choose colors that don't clash - works the same way in iOS.